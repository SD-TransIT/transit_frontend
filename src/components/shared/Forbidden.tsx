import React from 'react';

function Forbidden() {
  return (
    <div className="flex flex-col h-screen w-screen justify-center gap-3" data-testid="forbidden">
      <p className="text-3xl font-semibold mx-auto">403 - Forbidden</p>
      <p className="font-semibold  mx-auto">{'You don\'t have permission to access this page'}</p>
    </div>
  );
}

export default Forbidden;
