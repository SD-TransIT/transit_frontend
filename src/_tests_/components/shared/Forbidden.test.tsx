import { render, screen } from '@testing-library/react';
import React from 'react';
import Forbidden from '../../../components/shared/Forbidden';

describe('Unit tests for the Forbidden component.', () => {
  const FORBIDDEN_TEST_ID = 'forbidden';

  test('Should match snapshot.', () => {
    render(
      <Forbidden />,
    );
    expect(screen.getByTestId(FORBIDDEN_TEST_ID)).toMatchSnapshot();
  });
});
