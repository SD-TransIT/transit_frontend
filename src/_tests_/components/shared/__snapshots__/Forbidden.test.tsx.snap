// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for the Forbidden component. Should match snapshot. 1`] = `
<div
  class="flex flex-col h-screen w-screen justify-center gap-3"
  data-testid="forbidden"
>
  <p
    class="text-3xl font-semibold mx-auto"
  >
    403 - Forbidden
  </p>
  <p
    class="font-semibold  mx-auto"
  >
    You don't have permission to access this page
  </p>
</div>
`;
